ibr/ibr-sdk/jasmine/lib/jasmine-3.5.0/boot.js

```diff
@@ -0,0 +1,136 @@
+/**
```

Comment:
can we add jasmine through yarn instead of adding all of its source code? `yard add jasmine` should do the trick

Reply:
jasmine node does not support ES6 format. Replaced jasmine testing platform with AVA to allow ES6 testing.